lib/sign.js

```diff
@@ -24,7 +42,7 @@ function makeCert (publisherName, certFilePath, program) {
   // Inform the user about the password
   utils.log(chalk.green.bold('When asked to enter a password, please select "None".'))
 
-  return utils.executeChildProcess(makecert, makecertArgs)
+  return utils.executeChildProcess(makecertExe, makecertArgs)
```

Comment:
I renamed `makecert` -> `makecertExe` because I thought that was more clear (and avoids potential confusion from having a variable with the same name as the function containing it).